data/strings/strings.txt

```diff
@@ -15390,6 +15394,7 @@
     de = Geben Sie eine gültige LINE-Webadresse oder LINE ID ein
     es = Introduzca una dirección web o ID de LINE válidos
     et = Sisesta kehtiv LINE veebiaadress või LINE ID
+    fr = Entrez une adresse web ou une ID de LIGNE valide
```

Comment:
```suggestion
    fr = Entrez une adresse web ou un ID de LIGNE valide
```

Reply:
fr = Entrez une adresse web ou un ID de LINE valide

LINE is the name of the app no need to translate it